registrations/serializers.py

```diff
@@ -65,9 +71,26 @@ class SignUpSerializer(CreatedModifiedBaseSerializer):
         many=False,
         required=False,
     )
+    extra_info = serializers.CharField(required=False)
+    date_of_birth = serializers.DateField(required=False)
```

Comment:
I'm getting some errors when trying to use blank/null values is extra_info or date_of_birth fields.
For example `PUT http://linkedevents-backend:8080/v1/signup_group/5/` with payload
```suggestion
{
  "extra_info": "",
  "registration": 49,
  "signups": [
    {
      "city": "",
      "date_of_birth": null,
      "email": "jori.lindell@gmail.com",
      "extra_info": "",
      "first_name": "Jori",
      "id": 149,
      "last_name": "Lindell",
      "membership_number": "",
      "native_language": "fi",
      "notifications": "email",
      "phone_number": null,
      "responsible_for_group": true,
      "service_language": "fi",
      "street_address": null,
      "zipcode": null
    }
  ]
}
```
returns following errors:
```
{
  "signups": [
    {
      "date_of_birth": [
        "Tämän kentän arvo ei voi olla \"null\"."
      ],
      "extra_info": [
        "Tämä kenttä ei voi olla tyhjä."
      ]
    }
  ],
  "extra_info": [
    "Tämä kenttä ei voi olla tyhjä."
  ]
}
```

Should this be
```suggestion
    extra_info = serializers.CharField(required=False, allow_blank=True)
    date_of_birth = serializers.DateField(required=False, allow_null=True)
```